lambda/lambda_alarm_hive.py

```diff
@@ -69,10 +69,10 @@ def create_issue_for_account(accountId, excludeAccountFilter):
 
 def lambda_handler(event, context):
     debug = False
+    createHiveAlert = True
```

Comment:
Remove the hard-coded createHiveAlert = True line, since this variable is already defined based on an environment variable. Adjusting this value with it statically set also means you need to redeploy the Lambda function.

https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html